query_tests.py

```diff
@@ -30,24 +30,33 @@ def process_response(text):
     return result_dict
 
 
-def generate_filter(filter_name, udfs):
+def generate_filter(filter_name, udfs, distributed=False):
     log.info("Generating the filter %s with udfs %s", filter_name, udfs)
     cmd = f"{COMPILER_BINARY} "
     cmd += f"-q {QUERY_DIR.joinpath(filter_name)} "
     for udf in udfs:
         cmd += f"-u {UDF_DIR.joinpath(udf)} "
     cmd += "-r productpage-v1 "
+    if distributed:
+        cmd += f"-d true "
```

Comment:
-d true is an odd flag. Usually you only need -d to toggle an option. We should change this next time.